presto-orc/src/test/java/com/facebook/presto/orc/TestOrcBloomFilters.java

```diff
@@ -0,0 +1,333 @@
+        TupleDomain.ColumnDomain<TestingColumnHandle> column1 = new TupleDomain.ColumnDomain<>(testingColumnHandle1, testingColumnHandleDomain1);
+
+        // predicate consist of the bigint_0 = 1234
+        TupleDomain<TestingColumnHandle> effectivePredicate = TupleDomain.fromColumnDomains(Optional.<List<TupleDomain.ColumnDomain<TestingColumnHandle>>>of(ImmutableList.of(column)));
+        TupleDomain<TestingColumnHandle> emptyEffectivePredicate = TupleDomain.fromColumnDomains(Optional.<List<TupleDomain.ColumnDomain<TestingColumnHandle>>>of(ImmutableList.of()));
+
+        // predicate column references
+        ImmutableList.Builder<TupleDomainOrcPredicate.ColumnReference<TestingColumnHandle>> columnReferences = ImmutableList.builder();
+        columnReferences.add(new TupleDomainOrcPredicate.ColumnReference<>(testingColumnHandle, 0, BIGINT));
+        columnReferences.add(new TupleDomainOrcPredicate.ColumnReference<>(testingColumnHandle1, 1, BIGINT));
+
+        TupleDomainOrcPredicate predicate = new TupleDomainOrcPredicate<>(effectivePredicate, columnReferences.build(), true);
+        TupleDomainOrcPredicate emptyPredicate = new TupleDomainOrcPredicate<>(emptyEffectivePredicate, columnReferences.build(), true);
+
+        // assemble a matching and a non-matching bloom filter
+        HiveBloomFilter hiveBloomFilter = new HiveBloomFilter(new BloomFilter(1000, 0.01D));
+        OrcProto.BloomFilter emptyOrcBloomFilter = toOrcBloomFilter(hiveBloomFilter);
+        hiveBloomFilter.addLong(1234L);
+        OrcProto.BloomFilter orcBloomFilter = toOrcBloomFilter(hiveBloomFilter);
+
+        ImmutableMap.Builder<Integer, ColumnStatistics> matchingStatisticsBuilder = ImmutableMap.<Integer, ColumnStatistics>builder();
+        matchingStatisticsBuilder.put(0, new ColumnStatistics(null, null, new IntegerStatistics(10L, 2000L), null, null, null, null, new HiveBloomFilter(orcBloomFilter.getBitsetList(), orcBloomFilter.getBitsetCount() * 64, orcBloomFilter.getNumHashFunctions())));
+        Map<Integer, ColumnStatistics> matchingStatisticsByColumnIndex = matchingStatisticsBuilder.build();
+
+        ImmutableMap.Builder<Integer, ColumnStatistics> nonMatchingStatisticsBuilder = ImmutableMap.<Integer, ColumnStatistics>builder();
+        nonMatchingStatisticsBuilder.put(0, new ColumnStatistics(null, null, new IntegerStatistics(10L, 2000L), null, null, null, null, new HiveBloomFilter(emptyOrcBloomFilter.getBitsetList(), emptyOrcBloomFilter.getBitsetCount() * 64, emptyOrcBloomFilter.getNumHashFunctions())));
+        Map<Integer, ColumnStatistics> nonMatchingStatisticsByColumnIndex = nonMatchingStatisticsBuilder.build();
+
+        ImmutableMap.Builder<Integer, ColumnStatistics> withoutBloomFilterStatisticsBuilder = ImmutableMap.<Integer, ColumnStatistics>builder();
+        nonMatchingStatisticsBuilder.put(0, new ColumnStatistics(null, null, new IntegerStatistics(10L, 2000L), null, null, null, null, null));
```

Comment:
I'm pretty sure this is an typo and should be `withoutBloomFilterStatisticsBuilder`, but I'd use the simple form above